app/authentication/auth.py

```diff
@@ -36,12 +39,12 @@ def to_dict_v2(self):
 def get_auth_token(req):
     auth_header = req.headers.get('Authorization', None)
     if not auth_header:
-        raise AuthError('Unauthorized, authentication token must be provided', 401)
+        raise AuthError('Unauthorized: authentication token must be provided', 401)
```

Comment:
@idavidmcdonald Is in line with our style guide?

https://github.com/alphagov/notifications-manuals/wiki/A-to-Z-of-style#authorisation

Reply:
Yep, happy with this as it matches the http 401 error name of https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401